packages/core/content-releases/server/src/controllers/validation/release.ts

```diff
@@ -1,4 +1,4 @@
 import { validateYupSchema } from '@strapi/utils';
 import { RELEASE_SCHEMA } from '../../../../shared/validation-schemas';
 
-export const validateRelease = validateYupSchema(RELEASE_SCHEMA);
+export const validateRelease = validateYupSchema(RELEASE_SCHEMA.noUnknown());
```

Comment:
why is this added here and not on the actual schema where you also removed it prior? :)

Reply:
It was not working as all fields were not passed to BE, but its not the case now so reverted.